build-cross-tools.sh

```diff
@@ -78,7 +82,9 @@ if [ -n "$PYTHON" ]; then
     LLVM_ARGS="$LLVM_ARGS --with-python"
 fi
 
-./build-llvm.sh $PREFIX --host=$HOST $LLVM_ARGS
+if [ -z "$FULL_LLVM" ]; then
+    ./build-llvm.sh $PREFIX --host=$HOST $LLVM_ARGS
```

Comment:
This isn't right? This causes `build-llvm` to only be executed if `--full-llvm` wasn't specified.

You'd want to add this if condition around the call to `strip-llvm.sh`, just like in `build-all.sh`.

Reply:
Yes, this is just a mistake. Thanks.